VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -27,6 +30,10 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseComponent implemen
     showAudioRecordingAlert = false;
     expanedPanel = true;
 
+    consultationAccepted$: Subscription;
```

Comment:
the judge waiting room already subscribs to consultation messages (see base class)

Reply:
added getConsultationMessage ( in base class)